### PR TITLE
Search bar visible when having errors in WfoTableWithFilter

### PR DESCRIPTION
**Pull request by Georgi2704**
Screenshot 2024-08-16 at 15 05 22

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoError/WfoError.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 
 import { useTranslations } from 'next-intl';
+import { SerializableError } from 'next/dist/lib/is-serializable-props';
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoError/WfoError.tsx:4]: Is this import correct? Importing from a dist folder looks strange

File: packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx [hunk 2]
```diff
@@ -164,9 +164,10 @@ export const WfoSubscriptionsList: FC<WfoSubscriptionsListProps> = ({
         query: queryString || undefined,
     };
 
-    const { data, isFetching, isError } = useGetSubscriptionListQuery(
+    const { data, isFetching, error } = useGetSubscriptionListQuery(
         graphqlQueryVariables,
     );
+    // const serializedError: SerializedError = error ? error : { message: '' };
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx:170]: remove or uncomment

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx [hunk 3]
```diff
@@ -64,7 +65,7 @@ export type WfoTableWithFilterProps<T extends object> = {
     onUpdateQueryString: (queryString: string) => void;
     onUpdatePage: (criterion: Criteria<T>['page']) => void;
     onUpdateDataSort: (dataSorting: WfoDataSorting<T>) => void;
-    hasError?: boolean;
+    error?: RTKQueryError;
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx:68]: There can be errors that are not RTKQueryErrors I think so this typing isn't always correct right?

DISCUSS

**Review by DutchBen — APPROVED** [review]

**Review by wouter1975 — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx (hunk 3 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx]: fixed with the mapper

**Pull request merged**